scipy/_lib/_testutils.py

```diff
@@ -77,7 +77,7 @@ def __call__(self, label="fast", verbose=1, extra_argv=None, doctests=False,
 
         if doctests:
             # raise ValueError("Doctests not supported")
-            pytest_args += ["--doctest-modules", "--ignore=scipy/interpolate/_interpnd_info.py"]
+            pytest_args += ["--doctest-modules", "--ignore=scipy/interpolate/_interpnd_info.py", "--ignore=scipy/_lib/"]
```

Comment:
Minor: `_lib` ignore should be a tad more selective.

Reply:
It's because the `_lib` modules were being collected for doctesting on running: `python dev.py test --doctests -v`. I'm not sure if we want it included.
```
scipy/_lib/_disjoint_set.py::scipy._lib._disjoint_set.DisjointSet PASSED                                                                                                                             [  0%]
scipy/_lib/_docscrape.py::scipy._lib._docscrape.namedtuple FAILED                                                                                                                                    [  0%]
scipy/_lib/_finite_differences.py::scipy._lib._finite_differences.arange FAILED                                                                                                                      [  0%]
scipy/_lib/_finite_differences.py::scipy._lib._finite_differences.array FAILED                                                                                                                       [  0%]
scipy/_lib/_finite_differences.py::scipy._lib._finite_differences.hstack FAILED                                                                                                                      [  0%]
scipy/_lib/_finite_differences.py::scipy._lib._finite_differences.prod FAILED                                                                                                                        [  0%]
scipy/_lib/_gcutils.py::scipy._lib._gcutils.gc_state FAILED                                                                                                                                          [  0%]
scipy/_lib/_gcutils.py::scipy._lib._gcutils.assert_deallocated FAILED                                                                                                                                [  0%]
scipy/_lib/_tmpdirs.py::scipy._lib._tmpdirs.in_dir FAILED                                                                                                                                            [  0%]
scipy/_lib/_tmpdirs.py::scipy._lib._tmpdirs.in_tempdir FAILED                                                                                                                                        [  0%]
scipy/_lib/_tmpdirs.py::scipy._lib._tmpdirs.tempdir FAILED                                                                                                                                           [  0%]
scipy/_lib/_util.py::scipy._lib._util.AxisError FAILED                                                                                                                                               [  0%]
scipy/_lib/_util.py::scipy._lib._util.Generator PASSED                                                                                                                                               [  0%]
scipy/_lib/_util.py::scipy._lib._util.Generator.binomial FAILED                                                                                                                                      [  0%]
scipy/_lib/_util.py::scipy._lib._util.Generator.bytes FAILED                                                                                                                                         [  0%]
scipy/_lib/_util.py::scipy._lib._util.Generator.chisquare FAILED                                                                                                                                     [  1%]
scipy/_lib/_util.py::scipy._lib._util.Generator.choice FAILED                                                                                                                                        [  1%]
scipy/_lib/_util.py::scipy._lib._util.Generator.dirichlet FAILED                                                                                                                                     [  1%]
scipy/_lib/_util.py::scipy._lib._util.Generator.exponential FAILED                                                                                                                                   [  1%]
scipy/_lib/_util.py::scipy._lib._util.Generator.f FAILED                                                                                                                                             [  1%]
scipy/_lib/_util.py::scipy._lib._util.Generator.gamma FAILED                                                                                                                                         [  1%]
scipy/_lib/_util.py::scipy._lib._util.Generator.geometric FAILED                                                                                                                                     [  1%]
scipy/_lib/_util.py::scipy._lib._util.Generator.gumbel FAILED                                                                                                                                        [  1%]
scipy/_lib/_util.py::scipy._lib._util.Generator.hypergeometric FAILED                                                                                                                                [  1%]
scipy/_lib/_util.py::scipy._lib._util.Generator.integers FAILED                                                                                                                                      [  1%]
scipy/_lib/_util.py::scipy._lib._util.Generator.laplace FAILED                                                                                                                                       [  1%]
scipy/_lib/_util.py::scipy._lib._util.Generator.logistic FAILED                                                                                                                                      [  1%]
scipy/_lib/_util.py::scipy._lib._util.Generator.lognormal FAILED                                                                                                                                     [  1%]
scipy/_lib/_util.py::scipy._lib._util.Generator.logseries FAILED                                                                                                                                     [  1%]
scipy/_lib/_util.py::scipy._lib._util.Generator.multinomial FAILED                                                                                                                                   [  1%]
scipy/_lib/_util.py::scipy._lib._util.Generator.multivariate_hypergeometric FAILED                                                                                                                   [  2%]
scipy/_lib/_util.py::scipy._lib._util.Generator.multivariate_normal FAILED                                                                                                                           [  2%]
scipy/_lib/_util.py::scipy._lib._util.Generator.negative_binomial FAILED                                                                                                                             [  2%]
scipy/_lib/_util.py::scipy._lib._util.Generator.noncentral_chisquare FAILED                                                                                                                          [  2%]
scipy/_lib/_util.py::scipy._lib._util.Generator.noncentral_f FAILED                                                                                                                                  [  2%]
scipy/_lib/_util.py::scipy._lib._util.Generator.normal FAILED                                                                                                                                        [  2%]
scipy/_lib/_util.py::scipy._lib._util.Generator.pareto FAILED                                                                                                                                        [  2%]
scipy/_lib/_util.py::scipy._lib._util.Generator.permutation FAILED                                                                                                                                   [  2%]
scipy/_lib/_util.py::scipy._lib._util.Generator.permuted FAILED                                                                                                                                      [  2%]
scipy/_lib/_util.py::scipy._lib._util.Generator.poisson PASSED                                                                                                                                       [  2%]
scipy/_lib/_util.py::scipy._lib._util.Generator.power FAILED                                                                                                                                         [  2%]
scipy/_lib/_util.py::scipy._lib._util.Generator.random FAILED                                                                                                                                        [  2%]
scipy/_lib/_util.py::scipy._lib._util.Generator.rayleigh FAILED                                                                                                                                      [  2%]
scipy/_lib/_util.py::scipy._lib._util.Generator.shuffle FAILED                                                                                                                                       [  2%]
scipy/_lib/_util.py::scipy._lib._util.Generator.spawn FAILED                                                                                                                                         [  2%]
scipy/_lib/_util.py::scipy._lib._util.Generator.standard_cauchy FAILED                                                                                                                               [  3%]
scipy/_lib/_util.py::scipy._lib._util.Generator.standard_exponential FAILED                                                                                                                          [  3%]
scipy/_lib/_util.py::scipy._lib._util.Generator.standard_gamma FAILED                                                                                                                                [  3%]
scipy/_lib/_util.py::scipy._lib._util.Generator.standard_normal FAILED                                                                                                                               [  3%]
scipy/_lib/_util.py::scipy._lib._util.Generator.standard_t FAILED                                                                                                                                    [  3%]
scipy/_lib/_util.py::scipy._lib._util.Generator.triangular FAILED                                                                                                                                    [  3%]
scipy/_lib/_util.py::scipy._lib._util.Generator.uniform FAILED                                                                                                                                       [  3%]
scipy/_lib/_util.py::scipy._lib._util.Generator.vonmises FAILED                                                                                                                                      [  3%]
scipy/_lib/_util.py::scipy._lib._util.Generator.wald FAILED                                                                                                                                          [  3%]
scipy/_lib/_util.py::scipy._lib._util.Generator.weibull FAILED                                                                                                                                       [  3%]
scipy/_lib/_util.py::scipy._lib._util.Generator.zipf FAILED                                                                                                                                          [  3%]
scipy/_lib/decorator.py::scipy._lib.decorator.n_args PASSED                                                                                                                                          [  3%]
scipy/_lib/doccer.py::scipy._lib.doccer.docformat FAILED                                                                                                                                             [  3%]
scipy/_lib/doccer.py::scipy._lib.doccer.inherit_docstring_from FAILED                                                                                                                                [  3%]
scipy/_lib/doccer.py::scipy._lib.doccer.indentcount_lines FAILED                                                                                                                                     [  3%]
scipy/_lib/doccer.py::scipy._lib.doccer.unindent_string FAILED                                                                                                                                       [  4%]
scipy/_lib/uarray.py::scipy._lib.uarray.Dispatchable FAILED                                                                                                                                          [  4%]
scipy/_lib/uarray.py::scipy._lib.uarray.all_of_type FAILED                                                                                                                                           [  4%]
scipy/_lib/uarray.py::scipy._lib.uarray.determine_backend FAILED                                                                                                                                     [  4%]
scipy/_lib/uarray.py::scipy._lib.uarray.determine_backend_multi FAILED                                                                                                                               [  4%]
scipy/_lib/uarray.py::scipy._lib.uarray.generate_multimethod FAILED                                                                                                                                  [  4%]
scipy/_lib/uarray.py::scipy._lib.uarray.mark_as FAILED                                                                                                                                               [  4%]
scipy/_lib/_uarray/__init__.py::scipy._lib._uarray FAILED 
```